### PR TITLE
Upgrade to Blueprint v3

### PR DESCRIPTION
**Pull request by psiemens**
- Upgrade Blueprint from v1 to v3
- Improve styling throughout Dispatch admin interface

### CODE REVIEW

File: dispatch/static/manager/src/js/components/Header/Header.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import React from 'react'
 import { Link } from 'react-router'
 import LoadingBar from 'react-redux-loading-bar'
+import { Button } from '@blueprintjs/core'
+
 import HeaderButtons from './HeaderButtons'
 import MobileHeaderButtons from './MobileHeaderButtons'
-import { desktopSize } from '../../util/helpers'
+// import { desktopSize } from '../../util/helpers'
```
Comment (JamieRL) [inline dispatch/static/manager/src/js/components/Header/Header.js:8]: Should this still be here?

File: dispatch/static/manager/src/js/components/PageEditor/PageToolbar.js [hunk 2]
```diff
@@ -1,44 +1,56 @@
...
-          </AnchorButton>
-          <VersionsDropdown
-            current_version={props.page.current_version}
-            published_version={props.page.published_version}
-            latest_version={props.page.latest_version}
-            getVersion={props.getVersion} />
+      <ToolbarLeft>
+        <ul className='bp3-breadcrumbs'>
+          <li><Link to='pages' className='bp3-breadcrumb'>Pages</Link></li>
+          <li><span className='bp3-breadcrumb bp3-breadcrumb-current'>{props.isNew ? 'New page' : <Text ellipsize={true}>{props.pagep.title}</Text> }</span></li>
```
Comment (JamieRL) [inline dispatch/static/manager/src/js/components/PageEditor/PageToolbar.js]: typo: `props.pagep.title`

**Review by JamieRL — CHANGES_REQUESTED** [review]
Looks great. I really like the UI design. There are a few commented out sections that I'm not sure if we're keeping or not. There's also just 1 typo in the pagetoolbar that's causing errors

File: dispatch/static/manager/src/js/components/PageEditor/PageToolbar.js (hunk 2 above)
Comment (psiemens) [reply dispatch/static/manager/src/js/components/PageEditor/PageToolbar.js]: Thanks!

**Review by JamieRL — APPROVED** [review]

**Pull request merged**